index.html

```diff
@@ -4,11 +4,6 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
     <title>Gebiedsgericht Werken Dashboard</title>
-    <style>
```

Comment:
Isn't it safe to have a min-width? The application is not very responsive...

Reply:
nope, on mobile the user had to scroll horizontally to even see the dashboard. The grid takes care of the width